scripts/post-bump.sh

```
@@ -40,16 +41,61 @@ else
+      echo "Opening \$EDITOR ($EDITOR) for release notes..."
+      $EDITOR "$RELEASE_NOTES_FILE"
+    elif command -v nano >/dev/null 2>&1; then
+      echo "Opening nano editor for release notes..."
+      nano "$RELEASE_NOTES_FILE"
+    elif command -v vim >/dev/null 2>&1; then
+      echo "Opening vim editor for release notes..."
+      vim "$RELEASE_NOTES_FILE"
+    else
+      echo "Warning: No suitable editor found. Please edit $RELEASE_NOTES_FILE manually."
+      echo "Press Enter to continue after editing the file..."
+      read -r
```

Shellcheck (0.10.0)

[warning] In POSIX sh, read without a variable is undefined.

(SC3061)